plenum/test/replica/test_instance_faulty_processor.py

```diff
@@ -406,4 +407,16 @@ def test_process_backup_instance_faulty_msg_quorum_for_different_replicas(backup
     assert instance_to_remove not in backup_instance_faulty_processor.backup_instances_faulty
     assert len(node.replicas.remove_replica_calls) == 1
     assert node.replicas.remove_replica_calls[0] == instance_to_remove
-    assert nodes.issubset(backup_instance_faulty_processor.backup_instances_faulty[instance_not_removed].keys())
\ No newline at end of file
+    assert nodes.issubset(backup_instance_faulty_processor.backup_instances_faulty[instance_not_removed].keys())
+
+
```

Comment:
Please add an integration test reproducing the original problem from the bug

Reply:
ok